erc20-bridge-token/contracts/ProofConsumer.sol

```diff
@@ -81,17 +81,17 @@ contract ProofConsumer is Ownable, IProofConsumer {
                     .outcome
                     .executor_id
             ) == keccak256(nearTokenLocker),
```

Comment:
Maybe it is better to do all these checks at the begging of the function. I think we shouldn't store it in the `usedProofs` incorrect proofs.